e2e/test/scenarios/dashboard-cards/click-behavior.cy.spec.js

```diff
@@ -608,6 +672,646 @@ describe("scenarios > dashboard > dashboard cards > click behavior", () => {
       });
     });
   });
+
+  describe("table", () => {
+    const questionDetails = QUESTION_TABLE;
+    const dashboardDetails = {
+      parameters: [DASHBOARD_FILTER_TEXT],
+    };
+
+    it("should open drill-through menu as a default click-behavior", () => {
+      cy.createQuestionAndDashboard({ questionDetails }).then(
+        ({ body: card }) => {
+          visitDashboard(card.dashboard_id);
+        },
+      );
+
+      getTableCell(COLUMN_INDEX.COUNT).click();
+      popover().should("contain.text", "Filter by this value");
+
+      getTableCell(COLUMN_INDEX.CREATED_AT).click();
+      popover().should("contain.text", "Filter by this date");
+
+      editDashboard();
+
+      getDashboardCard().realHover().icon("click").click();
+      getDashboardCard()
+        .button()
+        .should("have.text", "Open the drill-through menu");
+    });
+
+    it("should allow setting dashboard and saved question as custom destination for different columns", () => {
+      cy.createQuestion(TARGET_QUESTION);
+      cy.createDashboard(
+        {
+          ...TARGET_DASHBOARD,
+          parameters: [DASHBOARD_FILTER_TEXT, DASHBOARD_FILTER_TIME],
+        },
+        {
+          wrapId: true,
+          idAlias: "targetDashboardId",
+        },
+      );
+      cy.createQuestionAndDashboard({ questionDetails }).then(
+        ({ body: card }) => {
+          visitDashboard(card.dashboard_id);
+        },
+      );
+
+      editDashboard();
+
+      getDashboardCard().realHover().icon("click").click();
+
+      (function addCustomDashboardDestination() {
```

Comment:
These IIFEs help with readability a bit (grouping, function name, indentation) and make the blocks foldable in the editor.